docs/admin/20-getting-started/20-docker-compose.md

```diff
@@ -5,84 +5,185 @@ title: Docker Compose
-- [**Git**](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
-- [**Docker Compose**](https://docs.docker.com/compose/install/)
+  Alternatively, you can use a wildcard domain (`*.YOUR.DOMAIN`)
+- A **hosted server** (e.g., Hetzner, AWS, or your own VPS) with Linux and SSH access
 
 ---
 
-##  1. Download
+## 1. Connect to Your Server
+Log into your server via SSH:
 
-Download the `opencloud_full` folder (this folder contains a multi-file Docker Compose configuration):
+```bash
+ssh root@YOUR.SERVER.IP
+```
 
-```Shell
-git clone https://github.com/opencloud-eu/opencloud.git
+## 2. Install Docker
+Update your system and install Docker.
+
+
+First, update your package list:
+
+```bash
+apt update && apt upgrade -y
 ```
+Install Docker following the [official Docker guide](https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository)
 
----
+Once Docker is installed, enable and start the service:
+
+```bash
+systemctl enable docker && systemctl start docker
+```
+
+## 3. Clone the OpenCloud Repository
+Download the necessary configuration files:
+
+```bash
+git clone https://github.com/opencloud-eu/opencloud.git
+```
 
-## 2. Start
+## 4. Configure the .env File for Staging Certificates
+Before requesting real SSL certificates, test the setup with Let's Encrypt’s staging environment.
 
-cd into the Docker Compose configuration folder:
+Navigate to the OpenCloud configuration folder:
 
-```Shell
+```bash
 cd opencloud/deployments/examples/opencloud_full
 ```
 
-Start the deployment with Docker Compose:
+Edit the `.env` file with the editor of your choice:
 
-```Shell
+In our example we use nano
+
+```bash
+nano .env
+```
+
+Modify these settings:
+
+### ✅ Disable insecure mode
+```bash
+# INSECURE=true
+```
+
+### ✅ Set your domain names
+```bash
+TRAEFIK_DOMAIN=traefik.YOUR.DOMAIN
+OC_DOMAIN=cloud.YOUR.DOMAIN
+COLLABORA_DOMAIN=collabora.YOUR.DOMAIN
+WOPISERVER_DOMAIN=wopiserver.YOUR.DOMAIN
+```
+
+### ✅ Set your admin password
+```bash
+ADMIN_PASSWORD=YourSecurePassword
+```
+
+### ✅ Set your email for SSL certification
+```bash
+TRAEFIK_ACME_MAIL=your@email.com
+```
+
+### ✅ Use Let's Encrypt staging certificates (for testing)
+```bash
+TRAEFIK_ACME_CASERVER=https://acme-staging-v02.api.letsencrypt.org/directory
+```
+
+Save and exit.
+
+## 5. Start OpenCloud
+Launch OpenCloud using Docker Compose:
+
+```bash
 docker compose up -d
 ```
 
-<img src={require("./img/quick-guide/quick-docker-compose-up.png").default} alt="Admin general" width="1920"/>
+This will start all required services in the background.
 
-This starts all necessary containers in the background.
+## 6. Verify SSL Certification
 
----
+In your web browser, visit:
+
+```bash
+https://cloud.YOUR.DOMAIN
+```
+
+You should see a security warning because the staging certificate is not fully trusted.
+Same should appear with the other domains you are using.
+
+Example with Chrome browser:
```

Comment:
Nitpicking: We promote open source - which Chrome is not particularly. Maybe consider using Firefox for these kind of things.